angular-ngrx-scss/src/app/user/services/user.service.ts

```diff
@@ -105,4 +108,49 @@ export class UserService {
       ),
     );
   }
+
+  getUserTopRepos(): Observable<UserReposState[]> {
+    const url = `${environment.githubUrl}/user/repos?sort=updated&affiliation=owner,collaborator,organization_member&per_page=20'}`;
```

Comment:
Can you add a note in the REST API doc we started to explain this url, please? I think this makes good sense for top repos, so we want to capture that info there for future showcases that use the REST API.

Link to the doc: https://docs.google.com/document/d/1qrAXPiMvag1ykFCd5dl0rLV3RtcapB9Dj8HVxrnXxLc/edit

- [ ] Update REST API doc to share this url and explain the decision to use these parameters